docs/README.md

```diff
@@ -1,60 +1,93 @@
-* `npm run docs:docker:build:production` - Run docker build for production environment.
-* `npm run docs:version <semver.version>` - Rollup a next version into `/<semver.version>/`.
-* `npm run docs:api` - Generates API References into `/next/api`.
-
-## Directory structure
-
-```
-- /next                         # The next version of documentation, unavailable on production build.
-    + /api                      # Automatically generated files from JsDoc. Do not edit!
-    *.md                        # The content of documentation
-+ /SEMVER.NUMBER                # Version of documentation, for instance 8.4, 8.5
-- /.vuepress                    # Vuepress files and configuration
-    - /containers               # Markdown containers `:::`
-        + /examples             # Container for running code snippets `::: example #exampleId .class :preset --html 1 --js 2`
-        sourceCodeLink.js       # Add link "Source Code" on the right, `::: source-code-link [URL]`
-    + /handsontable-manager     # This module provides api to run any example in any handsontable version with selected preset.
-    + /public                   # Public assets
-    + /theme                    # Theme overwrites and customizations
-    - /tools            
-        + /jsdoc-convert        # Tool used for converting jsdoc to markdown
-        + /version              # Tool used for rollup a new version
-        check-links.js          # Tool which check all links in the documentation
-        utils.js                # Utilities for tools
-    config.js                   # Configuration file
-    helpers.js                  # Common helpers to get sidebars and list of versions
-    highlight.js                # Code highlight configuration
-```
+# Handsontable documentation
+
+We treat documentation as an integral part of the Handsontable developer experience.
+
+That's why Handsontable comes with a vibrant, frequently-updated documentation portal.
+
+View the documentation's latest production version at [handsontable.com/docs](https://handsontable.com/docs).
+
+## Documentation overview
+
+The Handsontable documentation is made up of three main sections:
+- **Guides**: Carefully-written implementation guides.
+- **Examples**: Code examples covering real-life use cases.
+- **API reference**: API reference, automatically generated from the source code of each respective Handsontable version.
+
+We update the documentation:
+- With every Handsontable version release.
+- Whenever there's something new to add to improve.
+
+**See also:**
+
+* [Documentation editing guidelines](./README-EDITING.md)
+* [Documentation deployment guidelines](./README-DEPLOYMENT.md)
+
+## Getting started with Handsontable documentation
+
+To start a local Handsontable documentation server:
+
+1. From the `handsontable/docs` directory, install the documentation dependencies:
+    ```bash
+    npm install
+    ```
+2. From the `handsontable/docs` directory, generate the API reference:
```

Comment:
As the Readme is in the `docs` directory,  I don't think so, that repeating `From the `handsontable/docs` directory,` is required here.